example/offload-template/run-offload.cc

```diff
@@ -40,6 +71,26 @@ int main(int argc, char* argv[])
     // Initialize Celeritas
     auto& tmi = celeritas::TrackingManagerIntegration::Instance();
 
+    {
+        namespace fs = std::filesystem;
```

Comment:
Since this is an example, consider adding documentation, for example here, explaining that this relies on `SourceFiles.hh.in` being configured by CMake and resulting in the information being recorded, etc.